core/variant/callable.cpp

```diff
@@ -327,34 +328,47 @@ Callable::operator String() const {
 	}
 }
 
+Callable Callable::create(const Variant &p_variant, const StringName &p_method) {
+	ERR_FAIL_COND_V_MSG(p_method == StringName(), Callable(), "Method argument to Callable::create method must be a non-empty string.");
```

Comment:
maybe change `p_method == StringName()` to `p_method.is_empty()` for a better looking error message?

Reply:
The method does not exist in the `StringName` class.

Reply:
Wat?

What about this entry? https://docs.godotengine.org/en/stable/classes/class_stringname.html#class-stringname-method-is-empty

Reply:
This is GDScript only. See [source code](https://github.com/godotengine/godot/blob/master/core/string/string_name.h) instead.